source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,842 @@
+The update content that mixer generates consists of various pieces of OS
+content, update metadata, as well as a complete image. The OS content includes
+all files in an update, as well as zero- and delta-packs for improved update
+performance. The metadata, stored as manifests, describes all of the bundle
+information for the update. Update content produced by mixer is then published
+to a web server and consumed by clients via swupd. Refer to
+:ref:`swupd <swupd-guide>` for additional information regarding updates and
+update content.
 
-   .. code-block:: bash
-
-      sudo mkdir -p /var/www
+How it works
+************
 
-#. Create a symbolic link.
+Learn the mixer tool set up and workflow.
 
-   .. code-block:: bash
+.. contents::
+   :local:
+   :depth: 1
 
-      sudo ln -sf /home/clr/mix/update/www /var/www/mixer
+Prerequisites
+=============
 
-   .. note::
+* :command:`mixer` bundle
 
-      `/home/clr/mix` is the workspace for the mix (read `Create workspace`)
+  Add the mixer tool with the :command:`mixer` bundle. Refer to
+  `Install a bundle`_ for more details.
 
+* Docker container
 
-#. Set up ``nginx`` configuration. 
+  mixer by default runs all build commands in a Docker container to make sure
+  the correct tool versions are used. This also allows custom mixes to
+  automatically perform downstream format bumps when the upstream releases a
+  format bump. See `Format version`_ for additional information regarding
+  format bumps.
 
-   .. code-block:: bash
+  Refer to `Configure and enable Docker`_ for instruction.
 
-      sudo mkdir -p  /etc/nginx/conf.d
+* Docker proxy (optional)
 
-#. Copy the default example configuration file. 
+  If you use a proxy server, you must set your proxy environment variables and
+  create a proxy configuration file for the Docker daemon and container.
 
-   .. code-block:: bash
+  Consult your IT department if you are behind a corporate proxy for the correct
+  values.
 
-      sudo cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
+  Refer to `Configure Docker proxy info`_ for instruction.
 
-#. Open ``mixer.conf`` with an editor.
+* Location to host the update content and images
 
-   .. code-block:: bash
- 
-      sudo nano /etc/nginx/conf.d/mixer.conf 
+  In order for swupd to make use of your mix, the update content for your mix
+  must be hosted on a web server. Your mix will be configured with an update
+  location URL, which swupd will use to pull down updates.
 
-#. Add the server configuration content below to ``mixer.conf``. 
+  Refer to `Set up a nginx web server for mixer`_ for an simple example of
+  setting up an update location.
 
-   .. code-block:: console
+Mix setup
+==========
 
-      server {
-           server_name localhost;
-           location / {
-                     root /var/www/mixer;
-                     autoindex on; 
-           }
-      }
+Setup creates and initializes the workspace used by mixer. Complete
```

Comment:
This is probably a nit, but simply saying "Setup creates and initializes" to me seems like there's a Setup tool that does these steps. Maybe simpler to state "Set up the mix workspace with the following steps:"

Reply:
Thanks, @bktan8. Nit fixed.